cvat/apps/engine/media_extractors.py

```diff
@@ -661,12 +661,23 @@ def save_as_chunk(self, images, chunk_path):
                 ext = os.path.splitext(path)[1].replace('.', '')
                 output = io.BytesIO()
                 if self._dimension == DimensionType.DIM_2D:
-                    pil_image = rotate_within_exif(Image.open(image))
-                    pil_image.save(output, format=pil_image.format if pil_image.format else self.IMAGE_EXT, quality=100, subsampling=0)
+                    pil_image, rotated = rotate_within_exif(Image.open(image))
```

Comment:
I will recommend to have a separate method `is_image_rotated(img: Image)`. Or just open the image `img = Image.open(image)` and check rotation `img.getexif().get(ORIENTATION_EXIF_TAG) == ORIENTATION.NORMAL_HORIZONTAL. The code will be cleaner.